packages/engine-ui/src/utils/mapUtils.js

```diff
@@ -152,43 +154,43 @@ export const bookingIcon = (bookings) => {
 }
 
 export const bookingToFeature = (bookings) => {
-  return bookings.flatMap(({ id, pickup, delivery, status, route }) => {
-    const points = [
-      point([delivery.lon, delivery.lat], {
-        id,
-        properties: {
-          status,
-          color: '#ccffcc',
-        },
-      }),
-    ]
+  return bookings.flatMap(
+    ({ id, pickup, delivery, status, route, assigned_to }) => {
+      const points = [
+        point([delivery.lon, delivery.lat], {
+          id,
+          properties: {
+            status,
+            color: '#ccffcc',
+          },
+        }),
+      ]
 
-    if (status === 'assigned' && route) {
-      return [...points, routeAssignedToBooking({ id, route })]
-    }
+      if (status === 'assigned' && route) {
+        return [...points, routeAssignedToBooking({ id, route })]
+      }
 
-    if (route) {
-      return [
-        ...points,
-        line(
-          route.geometry.coordinates.map(({ lat, lon }) => [lon, lat]),
-          {
-            id,
-            properties: {
-              color: '#e6ffe6',
-              offset: 0,
-              address: { pickup: pickup, delivery: delivery },
-              type: 'booking',
-            },
-          }
-        ),
-      ]
+      if (route) {
+        return [
+          ...points,
+          line(
+            route.geometry.coordinates.map(({ lat, lon }) => [lon, lat]),
+            {
+              id,
+              properties: {
+                color: '#e6ffe6',
+                offset: 0,
+                address: { pickup: pickup, delivery: delivery },
+                type: 'booking',
+              },
+            }
+          ),
+        ]
+      }
+      return points
     }
-
-    return route ? points : []
-  })
+  )
```

Comment:
I'm not sure the change was made here but the bookings now show the "other point" that is not the parcel icon, at all times. I don't think that was intentional, right?

Reply:
No, just had to update the return value from the function so now you just see the parcel until you are on the details.